modules/psap-special-resource-operator-using-configmaps.adoc

```diff
@@ -0,0 +1,310 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+[id="deploy-simple-kmod-using-configmap-chart"]
+= Building and running the simple-kmod SpecialResource by using a ConfigMap
+
+In this example, the simple-kmod kernel module is used to show how SRO can manage a driver container which is defined in Helm chart templates stored in a ConfigMap.
+
+.Prerequisites
+
+* You have a running {product-title} cluster.
+* You set the Image Registry Operator state to Managed for your cluster.
+* You installed the OpenShift CLI (`oc`).
+* You are logged into the OpenShift CLI as a user with `cluster-admin` privileges.
+* You installed Node Feature Discovery (NFD) Operator.
+* You installed the Special Resource Operator.
+* You installed the Helm CLI (`helm`).
+
+.Procedure
+. To create a simple-kmod `SpecialResource` object, define an ImageStream and BuildConfig to build the image, and a ServiceAccount, Role, RoleBinding, and DaemonSet to run the container. The ServiceAccount, Role, and RoleBinding are required to run the DaemonSet with the privileged securityContext so that the kernel module can be loaded.
+.. Create a `templates` directory, and change into it:
++
+[source,terminal]
+----
+$ mkdir -p chart/simple-kmod-0.0.1/templates
+----
++
+[source,terminal]
+----
+$ cd chart/simple-kmod-0.0.1/templates
+----
+
+.. Save this YAML template for the ImageStream and BuildConfig in the `templates` directory as `0000-buildconfig.yaml`. Note that the templates such as `{{.Values.specialresource.metadata.name}}` are filled in by SRO, based on fields in the `SpecialResource` CR and variables known to the Operator such as `{{.Values.KernelFullVersion}}`
```

Comment:
I'd consider adding your "Note that" information in a callout that points to the these lines int he YAML below. You can always have a single callout (`<1>`) below the code block, and use it more than once on each line that is applicable. 

Also, just to clarify, because the SRO is filling them in, they should leave these lines alone and not change them at all, correct?